### PR TITLE
Halls can now be archived instead of deleted (not displayed in gui, s…

### PR DESCRIPTION
**Pull request by joha4270**
…upposed to be deleted for all intents and purposes, but won't crash on retrival of missing hall)

### CODE REVIEW

File: src/electric-mouse/Controllers/HallController.cs [hunk 1]
```diff
@@ -24,7 +24,10 @@ public HallController(ApplicationDbContext dbContext)
 
         public IActionResult Create()
         {
-            IQueryable<RouteHall> halls = _dbContext.RouteHalls.Include(s => s.Sections);
+            IQueryable<RouteHall> halls = _dbContext.RouteHalls
+                .Where(hall => hall.Archived == false)
+                .Include(s => s.Sections);
```
Comment (jakobbsm) [inline src/electric-mouse/Controllers/HallController.cs:29]: Order is .Include and then .Where / .First etc.

**Comment by joha4270** [comment]
Why?

As far as i can see, it produces the exact same SQL (barring a name), and
none of them sounds much clearer than the other when spoken out loud.

**Comment by jakobbsm** [comment]
It's about coding standards. The ordering makes sense in bigger queries, and being consistent is key. I'll leave it for now.

**Comment by jakobbsm** [comment]
Whoops

**Pull request merged**